Tests/Unit/NetworkServer/IotHubDeviceInfoConversionTests.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#nullable enable
+
+namespace LoRaWan.Tests.Unit.NetworkServer
+{
+    using global::LoraKeysManagerFacade;
+    using Newtonsoft.Json;
```

Comment:
Seems an import was missing that was needed for my [earlier suggestion](https://github.com/Azure/iotedge-lorawan-starterkit/pull/1382/files#r791650734) to compile:

```suggestion
    using Newtonsoft.Json;
    using Newtonsoft.Json.Linq;
```

Sorry.